src/scss/components/_buttons.scss

```diff
@@ -52,6 +56,9 @@
 	font-weight: 900;
 	font-size: 12px;
 	border: none;
+	&.wide {
```

Comment:
Rather than put this as an `&`, it would be much more useful as a general button class... That way you can use it regardless of the colour of the button. 

The same goes for line 31 `.full-width`. Why is named full width, if it is only 45% wide?

Why use either of these classes instead of the grid system that concise supplies for us?